systemd/systemd.go

```diff
@@ -965,20 +995,36 @@ loop:
 					allStopped = false
 				}
 			}
+
 			if allStopped {
 				return nil
```

Comment:
it's possible that we observe that services are stopped quicker than we handle the signal coming from the stop goroutine, so we should probably still wait for systemctl to complete